### DISCUSSION TITLE
PHP mailer characters issue (charset=iso-8859-1 ?)

### DISCUSSION THREAD

**Discussion by Mkbxl** [PHPMailer/PHPMailer#2916] [General]
hi
I hope somebody can help me regarding this characters issue with PHP mailer.
I've installed PP mailer PHPMailer 5.2.25 in several websites and it does not always work correctly regarding characters.

The email recipient is a GMAIL email address and the incoming messages that contain accents are correctly displayed.
Now when I use my own domain as a recipient, the accents (latin) are replaced by strange characters (example : Message: jÃ©jÃ©jÃ©).
I thought the problem was due to a misconfiguration from my domain provider, but it's not, they set everything correctly.

Now, when I turn the "automatic forward" from GMAIL to my domain, the same thing occurs : wrong characters display.
Strangly enough, when I MANUALLY forward the GMAIL messages to my domain, the characters are displayed correctly !

Now, if I save the GMAIL message to eml format and open it with stand alone outlook app, it won't display correctly either.

The only way for me to receive the messages correctly displayed is by using an online email service (GMAIL, MAIL.COM, LIVE) but this won't work when I use my personal email address from my domain.

I noticed the PHP mailer charset is "Content-Type: text/plain; charset=iso-8859-1" and I wonder if using the UTF-8 charset would help solve this problem?
But I don't know how to implement the UTF-8 string in the handler file.
Any suggestion would be really appreciated.
Thank you

**Comment by Synchro** [comment]
First thing, PHPMailer 5.2.25 is from the old 5.2 branch that has not been supported for many years. Update to the latest version.

To use UTF-8, first make sure that your PHP installation has the `mbstring` extension, then do this in your code:

    $mail->CharSet = 'UTF-8';

and then make sure that all of your actual content (in `Subject`, `Body`, etc, is actually in UTF-8. That's it.

**Reply by Synchro** [reply]
From the comment that you deleted, I would recommend that you contact the suppliers of that script and have them update it. PHP 7.4 itself is no longer receiving any updates, and nobody should be building new service using it. To see the config of your PHP (including the extensions it has loaded), put this in a `.php` file and load it through your web server:

```php
<?php phpinfo();
```
That will show you the full details of how it's configured (delete that file after you're finished with it!)

**Reply by Mkbxl** [reply]
thank you! I will definitely contact them
